Networking/NetworkingTests/Encoder/BlazeForecastedImpressionsInputEncoderTests.swift

```diff
@@ -0,0 +1,36 @@
+import XCTest
+@testable import Networking
+
+final class BlazeForecastedImpressionsInputEncoderTests: XCTestCase {
+
+    func test_blazeForecastedImpressionsInput_encoder_encodes_necessary_fields_correctly() throws {
+        // Given
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-MM-dd"
+        let startDate = dateFormatter.date(from: "2023-12-05")!
+        let endDate = dateFormatter.date(from: "2023-12-11")!
```

Comment:
Nit: you can use `try XCTUnwrap` to avoid forced unwrapping in unit tests.

Reply:
Thanks, added https://github.com/woocommerce/woocommerce-ios/pull/11573/commits/883f3083f5e3185bdfd8729fd9a413b970aaa9c3